docs/alerts/creating-experience-alerts.md

```diff
@@ -1,11 +1,39 @@
 title: Creating Experience Alerts
 description: Step-by-step alert creation instructions for Experience
 
-Experience App allows you to create alert rules based on [Apdex score](https://en.wikipedia.org/wiki/Apdex) of Page Loads, HTTP Requests and On-Page Transactions.
+Each Experience App metric that you can create alert rules on has a bell icon, when clicked, shows a dropdown menu listing the metrics that you can choose. 
```

Comment:
Each Experience App metric that you can create alert rules on has a bell icon, when clicked, shows a dropdown menu listing the metrics that you can choose. 
==> 
Each Experience App metric that you can create alert rules on has a bell icon that, when clicked, shows a dropdown menu listing the metrics that you can choose.

Reply:
Adjusted.